### ISSUE TITLE
Fix error handling code when processing the results from Google Civic Info API

### ISSUE THREAD

**Issue by qtrandev** [CodeForFtL/rep-ftl#4]
I had to comment out the error handling code to work here:
https://github.com/CodeForFtL/rep-ftl/blob/master/scripts/repsView.js#L173
but we need to put that back in to handle errors.

Note that we are using V2 of the Google Civic Information API now code other sections of the code needs to be fixed to handle this.
https://developers.google.com/civic-information/docs/v2/representatives/representativeInfoByAddress
https://github.com/CodeForFtL/rep-ftl/blob/master/scripts/repsView.js#L314

**Comment by qtrandev** [comment]
@daarons Interested in this item?

**Comment by daarons** [comment]
@qtrandev Yes, I'll take a look at it later though. I'm not feeling too good today I think I caught something and I have a test today.

**Comment by qtrandev** [comment]
@daarons Hope you're doing well David! Any time is fine. Just add a comment to what you think the problem can be.

**Comment by daarons** [comment]
@qtrandev I got it to work! I thought at first it was a javascript problem. Turns out it was a css problem! The `.dataContainer` class was set to `display: none`. I just removed the class from the div that holds `locationBlock` and it works perfectly. I didn't edit the css because I'm not sure if it would mess up the other two divs that are `.dataContainer`. 

See [here](https://github.com/daarons/rep-ftl/blob/gh-pages/repsView.html#L65)

I'm not sure how you want everything placed, or if you still want the "Loading..." text displayed, but it works! [link to my gh-pages](http://daarons.github.io/rep-ftl/) :+1:
